content/test/integration/controller/active-entity-by-content-hash.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { EnvironmentConfig } from '@katalyst/content/Environment'
+import { EntityId, Fetcher } from 'dcl-catalyst-commons'
+import fetch from 'node-fetch'
+import { loadStandaloneTestEnvironment } from '../E2ETestEnvironment'
+import { buildDeployData } from '../E2ETestUtils'
+import { TestServer } from '../TestServer'
+
+describe('Integration - Get Active Entity By Content Hash', () => {
+  const testEnv = loadStandaloneTestEnvironment()
+  let server: TestServer
+  const fetcher = new Fetcher()
```

Comment:
There is no need to create a `Fetcher`. The same file has a `fetchJson` function